sql/mysql_db/fbs/mysql_db.fbs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+table PrivilegeSetColumn {
+    name:string;
+    privs:[int];
+}
+
+table PrivilegeSetTable {
+    name:string;
+    privs:[int];
+    columns:[PrivilegeSetColumn];
+}
+
+table PrivilegeSetDatabase {
+    name:string;
+    privs:[int];
+    tables:[PrivilegeSetTable];
+}
+
+table PrivilegeSet {
+    global_static:[int];
+    //TODO: global_dynamic:[string];
+    databases:[PrivilegeSetDatabase];
+}
+
+// Entries in the user table
+table User {
+    user:string;
+    host:string;
+    privilege_set:PrivilegeSet;
+    plugin:string;
+    password:string;
+    password_last_changed:int64; // time.Time?
+    locked:bool;
+    //attributes:string; // *string?
```

Comment:
So since FlatBuffers don't support `*string` directly, you can mimic it with a struct. See https://stackoverflow.com/questions/53803880/flatbuffers-and-null-value

Reply:
Also, `password_last_changed` is fine as an `int64`.